src/Cron_Event_Command.php

```diff
@@ -150,11 +150,14 @@ public function list_( $args, $assoc_args ) {
 	 *     $ wp cron event schedule cron_test now hourly
 	 *     Success: Scheduled event with hook 'cron_test' for 2016-05-31 10:20:32 GMT.
 	 *
-	 *     # Schedule new cron event and pass associative arguments
-	 *     $ wp cron event schedule cron_test '+1 hour' --foo=1 --bar=2
+	 *     # Schedule new cron event and pass arguments
+	 *     $ wp cron event schedule cron_test '+1 hour' --0=first-parameter --1=second-parameter
```

Comment:
Using the same terminology everywhere makes it easier to understand.
```suggestion
	 *     $ wp cron event schedule cron_test '+1 hour' --0=first-argument --1=second-argument
```